microsoft/testsuites/performance/nestedperf.py

```diff
@@ -252,10 +257,11 @@ def perf_nested_kvm_ntttcp_private_bridge(
             )
         finally:
             # stop running QEMU instances
-            node.tools[Qemu].delete_vm()
+            # node.tools[Qemu].delete_vm()
```

Comment:
If it's not needed, remove the code.

Reply:
Forgot to remove comment. Will fix it in next push